Content.Server/Botany/Systems/PlantAnalyzerSystem.cs

```diff
@@ -0,0 +1,261 @@
+            seedChem, plantHarvestType, exudeGases, mutationsList, trayChecker, tolerancesList, generalTraitsList, scanMode);
+    }
+
+    /// <summary>
+    ///     Returns information about the seed.
+    /// </summary>
+    public List<string> CheckGeneralTraits(SeedData plant)
+    {
+        var name = Loc.GetString(plant.DisplayName);
+        float endurance = plant.Endurance;
+        float yield = plant.Yield;
+        float lifespan = plant.Lifespan;
+        float maturation = plant.Maturation;
+        float growthStages = plant.GrowthStages;
+        float potency = plant.Potency;
+        return new List<string> { name, endurance.ToString(), yield.ToString(), lifespan.ToString(), maturation.ToString(), growthStages.ToString(), potency.ToString() };
+    }
+
+    public List<string> CheckAllMutations(SeedData plant)
+    {
+        string plantMutations = "";
+        string plantSpeciation = "";
+        plantSpeciation += String.Join(", \n", plant.MutationPrototypes.Select(item => item.ToString()));
+
+        if (plant.TurnIntoKudzu) plantMutations += "\n   " + $"{Loc.GetString("plant-analyzer-mutation-turnintokudzu")}";
+        if (plant.Seedless) plantMutations += "\n   " + $"{Loc.GetString("plant-analyzer-mutation-seedless")}";
+        if (plant.Slip) plantMutations += "\n   " + $"{Loc.GetString("plant-analyzer-mutation-slip")}";
+        if (plant.Sentient) plantMutations += "\n   " + $"{Loc.GetString("plant-analyzer-mutation-sentient")}";
+        if (plant.Ligneous) plantMutations += "\n   " + $"{Loc.GetString("plant-analyzer-mutation-ligneous")}";
+        if (plant.Bioluminescent) plantMutations += "\n   " + $"{Loc.GetString("plant-analyzer-mutation-bioluminescent")}";
+        if (plant.CanScream) plantMutations += "\n   " + $"{Loc.GetString("plant-analyzer-mutation-canscream")}";
+        return new List<string> { plantMutations, plantSpeciation };
+    }
+
+    public List<string> CheckAllTolerances(SeedData plant)
+    {
+        var nutrientConsumption = plant.NutrientConsumption;
+        var waterConsumption = plant.NutrientConsumption;
+        var idealHeat = plant.IdealHeat;
+        var heatTolerance = plant.HeatTolerance;
+        var idealLight = plant.IdealLight;
+        var lightTolerance = plant.LightTolerance;
+        var toxinsTolerance = plant.ToxinsTolerance;
+        var lowPresssureTolerance = plant.LowPressureTolerance;
+        var highPressureTolerance = plant.HighPressureTolerance;
+        var pestTolerance = plant.PestTolerance;
+        var weedTolerance = plant.WeedTolerance;
+        return new List<string> { nutrientConsumption.ToString(), waterConsumption.ToString(), idealHeat.ToString(), heatTolerance.ToString(), idealLight.ToString(),
+            lightTolerance.ToString(), toxinsTolerance.ToString(),lowPresssureTolerance.ToString(),highPressureTolerance.ToString(),pestTolerance.ToString(), weedTolerance.ToString() };
+    }
```

Comment:
This method is just a mess, you shouldn't just be randomly indexing into this to get data out.